tests/e2e/containers/containers.go

```diff
@@ -106,7 +106,7 @@ func (m *Manager) ExecCmd(t *testing.T, chainId string, validatorIndex int, comm
 			}
 
 			if success != "" {
-				return strings.Contains(outBuf.String(), success) || strings.Contains(errBuf.String(), success)
+				return strings.Contains(outBuf.String(), success) || strings.Contains(errBufString, success)
```

Comment:
The only problem I have with this is, wouldn't this not trigger if the error said something like "this tx was not successful" since the string contains success?

Reply:
Oh wait my bad, thats not the case

Reply:
but this is exactly as it used to be. In some cases, we expect stderr to containers success results. That's why compare stderr against success

Also note that we are comparing against `success` variable, not "success" string

Reply:
I think I accidentally duplicated that success code in my very first commit. Sorry if that was confusing

Reply:
Yeah this is acceptable, I misread it on first pass